.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: julia
+os:
+  - linux
+  - osx
+julia:
+  - 0.4
+  - nightly
+notifications:
+  email: false
+sudo: false
+cache:
+  directories:
+  - $HOME/usr
+addons:
+  apt_packages:
+  - gfortran
+env:
+  - MAKEFLAGS="-j3"
+    LD_LIBRARY_PATH=$HOME/usr/lib
```

Comment:
I copied from CoinOptServices.jl and didn't try without it. I presume it's so the packages can find the cached libraries and not bulld them again? cc @tkelman